dump25/r_28.py

```
@@ -0,0 +1,330 @@
+    # ---
+    if not linesc:
+        return
+    # ---
+    dump_done["claims"] += 1
+    # ---
+    items_file = dump_dir_claims / f"{dump_done['claims']}.json"
+    # ---
+    with open(items_file, "w", encoding="utf-8") as f:
+        json.dump(linesc, f)
+    # ---
+    fixed = {}
```

Ruff (0.8.2)

Local variable `fixed` is assigned to but never used

Remove assignment to unused variable `fixed`

(F841)